app/assets/stylesheets/styles/_presets.css.scss

```diff
@@ -143,6 +148,25 @@ textarea {
   float: none;
 }
 
+// Must reside in the top level of stylesheets
+// Note: sticky-footer is setting the height of the first selector to auto !important. To counteract that we are setting it manually.
+@include sticky-footer($footer-height, "#foo-bar", "#layout_footer", "#footer");
+#content {
```

Comment:
@warpling - let me know what you want to switch off. Hound just kind of turned itself on for CSS commenting, and I've let it run, but it may not all be relevant.

I think the // comments make sense though - they don't make their way into the compiled CSS code.